k8s/helm/component-values-metadata.yaml

```
@@ -0,0 +1,13 @@
+# This file contains mapping information for log levels of this component
+apiVersion: v1
+metavalues:
+  mainLogLevel:
+    name: Log-Level
+    description: The central configuration value to set the log level for this component
+    keys:
+      - path: promtail.server.config.log_level
```

```suggestion
      - path: promtail.config.logLevel
```
match the changed path in the values.yaml